amazon-mws.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
 
   s.add_dependency "ruby-hmac", "~> 0.4.0"
-  s.add_dependency "roxml", "~> 3.3.1"
+  s.add_dependency "roxml", ">= 3.3.1"
```

Comment:
🤔 probably better to indicate this dependency as "~> 4.0.0" since support for ruby 2.4 was added in 4.0.0

Reply:
sure